apps/labs/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+October 17, 2023
+
+Via Storyblok
+
+- Remove Thomas Fan and Pamphile Roy from the team page
+  (switch Role from `team` to `author`)
```

Comment:
```suggestion
- Remove Thomas Fan, Mars Lee, and Pamphile Roy from the team page (switch Role from `team` to `author`)
- Added Mateusz Sokół (switch Role from `author` to `team`) and Isuru Fernandoto (newly added) the team page
  
```

Reply:
@bskinn - Thanks! I'll commit this update (that I made) so we can merge and start the release process :)